app/src/main/java/com/vero/tasky/agenda/domain/model/Photo.kt

```diff
@@ -0,0 +1,6 @@
+package com.vero.tasky.agenda.domain.model
+
+data class Photo (
```

Comment:
As mentioned on another comment, you have 2 type of photos

Local Photos (photos that werent uploaded yet) only have uris (path to where they are located on the phone?
Remote Photos (local photos that were successfuly uploaded to the server, now have a public url for anyone to access)